subprojects/docs/src/docs/userguide/jvm/java_plugin.adoc

```diff
@@ -503,7 +503,11 @@ To help you understand how incremental compilation works, the following provides
 
 * If you are using an annotation processor that reads resources (e.g. a configuration file), you need to declare those resources as an input of the compile task.
 * If a resource file is changed, Gradle will trigger a full recompilation.
-* Using a custom `executable` or `javaHome` deactivates some optimizations. The compile task does not use incremental build immediately after a compile error or if a Java constant changes. Use <<toolchains.adoc#toolchains, toolchains>> instead if possible.
+* Using a custom `executable` or `javaHome` deactivates some optimizations.
+The compile task does not use incremental build immediately after a compile error or if a Java constant changes.
+Use <<toolchains.adoc#toolchains, toolchains>> instead if possible.
```

Comment:
```suggestion
  The compile task does not use incremental build immediately after a compile error or if a Java constant changes.
  Use <<toolchains.adoc#toolchains, toolchains>> instead if possible.
```

Should we put this sentence in the bullet point? Right now I think it'll render as a separate paragraph (not sure without a rendered docs build though). The suggestion above *should* add it to the bullet point.

Reply:
I verified the rendering. New lines are not block breaks in asciidoctor. So this is OK.
Note that it comes from our desire to have one sentence per line, which makes diffs much more readable in general.

Reply:
Gotcha, I guess I've never tested this since most markdown formats don't allow you to do it. Carry on then!

Reply:
Actually, thanks for the comment. Made me check.
In other places in the file, we actually use 4 spaces to mark that the sentence is inside the node. And it does not mess with rendering, so updated here as well.